features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -71,10 +71,16 @@ const Header = styled.header`
   }
 `;
 
-const Logo = styled.img`
+const Logo = styled.img<{ isCollapsed: boolean }>`
   width: 7.375rem;
+  content: url("/icons/logo-large.svg");
```

Comment:
It's actually that easy. Hands down the best solution that I've seen so far 🚀 My own implementation was far more complicated than this.

Reply:
thanks, that's great to hear :)